### PR TITLE
update and AEX-2 adjustments

### PR DESCRIPTION
**Pull request by thepiwo**
still WIP draft

### CODE REVIEW

**Comment by thepiwo** [comment]
we still need to wait for sdk 7.3.1 release

File: src/App.vue [hunk 1]
```diff
@@ -32,27 +32,32 @@
       }
     },
     async created() {
+      //First try AEX-2
```
Comment (kenodressel) [inline src/App.vue:35]: why do we need to first try AEX-2 if we laready have a wallet? (eg. foundWallet is true?)

File: src/utils/walletSearch.js [hunk 2]
```diff
@@ -50,12 +51,20 @@ export const wallet = {
...
       name: 'AEPP',
-      nodes: [{ name: 'test-net', instance: await Node({ url: NODE_URL, internalUrl: NODE_INTERNAL_URL }) }],
-      compilerUrl: COMPILER_URL
+      nodes: [
+        {name: 'ae_uat', instance: await Node({url: TESTNET_URL})},
+        {name: 'ae_mainnet', instance: await Node({url: MAINNET_URL})}
+      ],
+      compilerUrl: COMPILER_URL,
+      onNetworkChange (params) {
+        this.selectNode(params.networkId); // params.networkId needs to be defined as node in RpcAepp
```
Comment (kenodressel) [inline src/utils/walletSearch.js:63]: how does this work if the wallet tries to switch to a custom network?

File: src/App.vue (hunk 1 above)
Comment (thepiwo) [reply src/App.vue:35]: `foundWallet` if false when `created` is called

File: src/utils/walletSearch.js (hunk 2 above)
Comment (thepiwo) [reply src/utils/walletSearch.js:63]: doesn't work with AEX-2, this is a known issue in the specification

File: src/App.vue (hunk 1 above)
Comment (kenodressel) [reply src/App.vue:35]: I know we had issues with the app being re-rendered in the past but I assume we can attend to them, when they appear again.

**Pull request merged**